src/app/pages/virtualization/components/all-instances/instance-list/instance-list.component.html

```diff
@@ -1,5 +1,16 @@
 <div class="container">
   <div class="table-container">
+    <div class="table-header">
+      <h2>{{ 'Instances' | translate }}</h2>
+
+      @if (checkedInstances?.length > 0) {
+        <ix-instance-list-bulk-actions
```

Comment:
We may want the same for the `installed apps`, or even make a one reusable component for both. At this moment I only improved code to have separated component for instances bulk list actions.

<img width="1004" alt="Screenshot 2024-11-21 at 14 35 26" src="https://github.com/user-attachments/assets/c3928d5a-e431-4fe0-b042-5bbe19927c01">

Reply:
@undsoft - you may want to create a ticket for this suggestion?